### PR TITLE
2256 docs for arpa worker and queue

### PR DESCRIPTION
**Pull request by tzinckgraf**
## Description
Added some docs changes and script changes to get queues working for ARPA. This will help test ARPA locally using the docker / localstack setup.

## Testing
This can be tested by running `docker-compose up`, `yarn start:arpa-audit-worker` and running an audit report, which should use the queue.
This changes does not setup the arpa audit worker in docker.

### Automated and Unit Tests
- [ ] Added Unit tests

### Manual tests for Reviewer
- [ ] Added steps to test feature/functionality manually

## Checklist
- [x] Provided ticket and description
- [ ] Provided screenshots/demo
- [x] Provided testing information
- [ ] Provided adequate test coverage for all new code
- [x] Added PR reviewers

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ services:
     container_name: "gost-localstack_main"
     image: localstack/localstack
     ports:
-      - 4566:4566           # LocalStack Gateway
+      - 4566:4566           # SQS
```
Comment (TylerHendrickson) [inline docker-compose.yml:32]: Small nit, but I recommend reverting this change in the spirit of maintaining accuracy and future clarity. Port 4566 is LocalStack's gateway port through which most (all?) service requests are delivered, and is not exclusive to SQS.

File: docker/README.md [hunk 2]
```diff
@@ -163,3 +163,26 @@ any files within this directory will be ignored by git. Simply update the `postg
...
+bash-5.0# awslocal sqs list-queues
+```
+Now we can use that queue name and set it as the environment variable `ARPA_AUDIT_REPORT_SQS_QUEUE_URL`.
+
+To check that you can access the queues locally, you can use the `aws cli`
+```
+aws --region us-west-2 --endpoint http://localhost:4566 sqs list-queues
+```
+For this to work, you must have `AWS_ACCESS_KEY_ID=test` and `AWS_SECRET_ACCESS_KEY=test`.
+This is part of the standard localstack setup.
```
Comment (TylerHendrickson) [inline docker/README.md:188]: Suggest adding a note somewhere in here that the `localstack/entrypoint/init-aws.sh` script is executed automatically when the Docker container spins up, manual queue creation using `docker exec` / `awslocal` _may_ not be necessary.

File: packages/server/.env.example [hunk 3]
```diff
@@ -19,10 +19,10 @@ TEST_EMAIL_RECIPIENT=""
...
 
-# AWS SES:
-AWS_ACCESS_KEY_ID=
-AWS_SECRET_ACCESS_KEY=
-AWS_DEFAULT_REGION="${AWS_DEFAULT_REGION:-us-west-2}"
+# AWS SES. test / test is used for localstack
+AWS_ACCESS_KEY_ID=test
+AWS_SECRET_ACCESS_KEY=test
+AWS_DEFAULT_REGION="us-west-2"
 NOTIFICATIONS_EMAIL=grants-identification@usdigitalresponse.org
```
Comment (TylerHendrickson) [inline packages/server/.env.example:26]:
```suggestion
# AWS SDK configuration. test / test is used for localstack
AWS_ACCESS_KEY_ID=test
AWS_SECRET_ACCESS_KEY=test
AWS_DEFAULT_REGION="us-west-2"

# AWS SES
NOTIFICATIONS_EMAIL=grants-identification@usdigitalresponse.org
```

**Review by TylerHendrickson — CHANGES_REQUESTED** [review]

File: docker/README.md (hunk 2 above)
Comment (tzinckgraf) [reply docker/README.md:188]: Done

**Review by TylerHendrickson — APPROVED** [review]

**Review by TylerHendrickson — APPROVED** [review]

**Pull request merged**